web/src/components/landing-header.js

```diff
@@ -0,0 +1,113 @@
+import {Container,
+        Grid,
+        Typography
+} from '@material-ui/core';
+import React from 'react';
+import LandiingViewButton from './landing-view-button';
+import { makeStyles } from '@material-ui/core/styles';
+
+const landing_bg = require('../assets/landing-bg.jpg');
+const benefit = [
+  require('../assets/benefit1.png'),
+  require('../assets/benefit2.png'),
+  require('../assets/benefit3.png')
+];
+
+// Main Header css style
+const mainHeaderStyles = makeStyles(theme => ({
+  root: {
+    alignItems: 'center',
+    backgroundImage: `url(${landing_bg})`,
+    backgroundSize: 'cover',
+    color:'white',
+    height: 400,
+    justifyContent: 'center',
+    textAlign: 'center',
+    width: '100%'
+  },
+}));
+
+// Sub Explanation css style
+const subExplanationStyles = makeStyles(theme => ({
+  root: {
+    marginTop: 100,
+    marginBottom: 50,
+    marginLeft: 20,
+    marginRight: 20
+  },
+  img: {
+    width: 100,
+    height: 100
+  },
+  text: {
+    textAlign: 'center'
+  }
+}));
+
+// Sub Explanation in Landing Page
+function LandingSubExplanation() {
+  const subStyle = subExplanationStyles();
+  return (
+    <div className={subStyle.root}>
+      <Grid container justify="space-between" className={subStyle.text}>
+        <Grid item xs='auto' sm={3}>
+          <img src={benefit[0]} className={subStyle.img} />
+          <Typography variant="h6" gutterBottom>
+            Learn from a guru
+          </Typography>
+          <Typography>
+            Lorem ipsum door sit amet, consectetur adipiscing
+            elit, sed do elusmo
+          </Typography>
+        </Grid>
+        <Grid Item xs='auto' sm={3}>
+          <img src={benefit[1]} className={subStyle.img} />
+          <Typography variant="h6" gutterBottom>
+            Read with like-minded friends
+          </Typography>
+          <Typography>
+            Lorem ipsum door sit amet, consectetur adipiscing
+            elit, sed do elusmo
+          </Typography>
+        </Grid>
+        <Grid item xs='auto' sm={3}>
+          <img src={benefit[2]} className={subStyle.img} />
+          <Typography variant="h6" gutterBottom>
+            Read, drink and be merry
+          </Typography>
+          <Typography>
+            Lorem ipsum door sit amet, consectetur adipiscing
+            elit, sed do elusmo
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  )
+}
+
+// Header in Landing Page including main & sub
```

Comment:
Usually, header refers to the top section of this page, which is 'Tiny Social'. This section can be renamed to Body Summary? This is not the best name, so feel free to give this a thought.

Reply:
I think that 'Landing Description' is good. so I separated Main Landing Description & Sub Landing Description. what do you think about?